e2e/Pages/MapComponenet.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from '@playwright/test';
+import { Locator, Page } from 'playwright';
+
+import DashboardPage from './DashboardPage';
+
+class MapComponenet extends DashboardPage {
```

Comment:
typo: `Componenet` should be `Component`

Reply:
Fixed